frontend/DataView/DataView.js

```diff
@@ -74,9 +74,20 @@ class DataView extends React.Component<DataViewProps> {
       return <div style={missingStyle(theme)}>null</div>;
     }
 
+    const dataType = typeof data;
+    var isSingleValue = dataType === 'number' || dataType === 'string' || dataType === 'boolean';
```

Comment:
I kind of prefer "simple type" rather than "single value" but this is just a subject nit